src/main/webapp/css/main.css

```diff
@@ -62,6 +62,30 @@ body {
   display: none;
 }
 
+#conversationContainer{
+  padding-top: 50px;
+  padding-left: 50px;
+  width: 600px;
+}
+
+.card{
+  overflow-x: scroll;
+  max-height: 500px;
```

Comment:
nit: It's better to have this as a percentage than a fixed value, otherwise we're not making use of larger screens.